src/test/java/org/retest/rebazer/RepositoryHostingTypesTest.java

```diff
@@ -0,0 +1,50 @@
+package org.retest.rebazer;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import org.junit.Test;
+import org.retest.rebazer.connector.BitbucketConnector;
+import org.retest.rebazer.connector.GithubConnector;
+import org.retest.rebazer.connector.RepositoryConnector;
+import org.retest.rebazer.domain.RepositoryConfig;
+import org.springframework.boot.web.client.RestTemplateBuilder;
+
+public class RepositoryHostingTypesTest {
```

Comment:
I think we should explicitly add JUnit (Vintage and Jupiter) dependencies and convert new tests into JUnit 5.